src/components/TaskHeader.js

```diff
@@ -1,123 +1,30 @@
-import React, {useEffect} from 'react';
```

Comment:
I think we would remove this file completely actually, the only thing this is doing is showing the call to action button right? This will also be needed for manual requests, so maybe we can merge this into Header and make it a little more generalized

Reply:
done